React/wbooks/src/Navbar/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import logoutActions from '../redux/logout/actions';
+
+import NavBar from './layout';
+
+class NavbarContainer extends Component {
+  state = { isMenuDisplayed: false, areNotificationsDisplayed: false };
+
+  toogleMenu = () => {
+    this.setState({ isMenuDisplayed: !this.state.isMenuDisplayed });
+  };
+
+  toogleNotif = () => {
```

Comment:
Just for convention reasons, you should name this handle[Action], so in this case handeToggleNotification